src/node-results.js

```diff
@@ -90,8 +90,11 @@ class TableFull extends Component {
       // set first col to field name
       const firstCol = field;
       // default second col to field value in node
-      let secondCol =
-        this.props.node[field] || this.props.node.data[field] || '';
+      let secondCol = this.props.node[field];
+      if (secondCol === undefined)
+        secondCol = this.props.node.data[field];
+      if (secondCol === undefined)
```

Comment:
Why this `if` is identical to the one on line 96?

Reply:
First it checks if it can find the property under the main/first level of the `node` object. Then, if it can't, it tries to find that property under the `data` property of the `node` object. Then finally, if it also can't find it there, it sets it to `''`

Reply:
Thanks. I got it.